app/views/docs/query.phtml

```diff
@@ -0,0 +1,276 @@
+<p>
+Many Appwrite endpoints allows you to filter returned results through queries. Appwrite's SDKs include a <span class="tag">Query</span> class that helps construct query strings. This page documents all supported operations for Client SDKs. Server SDKs will behave similarly. 
+</p>
+
+<h2><a href="/docs/query#equal" id="equal">Equal</a></h2>
+<p>
+Returns results where the specified <b>attribute</b> is exactly equal to <b>value</b>.
+</p>
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Web</h3>
+        <div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Query } from "appwrite";
+const queryString = Query.equal(attribute, value);
+console.log(queryString);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Flutter</h3>
+        <div class="ide" data-lang="javascript" data-lang-label="Flutter SDK">
+        <pre class="line-numbers"><code class="prism language-dart" data-prism>import 'package:appwrite/appwrite.dart';
+String queryString = Query.equal(attribute, value);
+print(queryString);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Android</h3>
+        <div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import 'package:appwrite/appwrite.dart';
+String queryString = Query.equal(attribute, value)
+print(queryString)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>iOS</h3>
+        <div class="ide" data-lang="swift" data-lang-label="Apple SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+let queryString = Query.equal(attribute, value)
+print(queryString)</code></pre>
+        </div>
+    </li>
+</ul>
+
+<h2><a href="/docs/query#not-equal" id="not-equal">Not Equal</a></h2>
+<p>
+Returns results where the specified <b>attribute</b> is not equal to <b>value</b>.
```

Comment:
In all these examples , we should mention that value can either be a single value like `Avengers` or a list of values like `['Avengers', 'Titanic', 'Iron Man']`

Reference : https://github.com/appwrite/sdk-for-web/blob/master/src/sdk.ts#L3748-L3767

Reply:
You can use a numerical example to make it more clear